Uh this fails to build on x86:

```
error[E0308]: mismatched types
   --> /Users/matthias/.cargo/git/checkouts/zed-21fdc7b373199252/f4a995c/crates/gpui/src/platform/mac/window.rs:419:16
    |
419 |             if !self.native_window.isVisible() {
    |                ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ expected `bool`, found `i8`

For more information about this error, try `rustc --explain E0308`.
error: could not compile `gpui` (lib) due to 1 previous error
warning: build failed, waiting for other jobs to finish...
```

Not sure why the types are different between arm64 and x86.

--- a/crates/gpui/src/platform/mac/window.rs
+++ b/crates/gpui/src/platform/mac/window.rs
@@ -415,6 +415,15 @@ impl MacWindowState {
 
     fn start_display_link(&mut self) {
         self.stop_display_link();
+        unsafe {
+            if !self
+                .native_window
+                .occlusionState()
+                .contains(NSWindowOcclusionState::NSWindowOcclusionStateVisible)
+            {
+                return;
+            }
+        }
         let display_id = unsafe { display_id_for_screen(self.native_window.screen()) };
         if let Some(mut display_link) =
             DisplayLink::new(display_id, self.native_view.as_ptr() as *mut c_void, step).log_err()

